bundles/binding/org.openhab.binding.insteonplm/src/main/java/org/openhab/binding/insteonplm/internal/driver/IOStream.java

```diff
@@ -103,21 +102,21 @@ private synchronized boolean reconnect() {
 
     /**
      * Creates an IOStream from an allowed config string:
-     * 
+     *
      * /dev/ttyXYZ (serial port like e.g. usb: /dev/ttyUSB0 or alias /dev/insteon)
-     * 
+     *
      * /hub2/user:password@myinsteonhub.mydomain.com:25105,poll_time=1000 (insteon hub2 (2014))
-     * 
+     *
      * /hub/myinsteonhub.mydomain.com:9761
-     * 
+     *
      * @param config
      * @return reference to IOStream
      */
 
     public static IOStream s_create(String config) {
         if (config.startsWith("/hub2/")) {
             return makeHub2014Stream(config);
-        } else if (config.startsWith("/hub/")) {
+        } else if (config.startsWith("/hub/") || config.startsWith("/tcp/")) {
```

Comment:
Could you update the JavaDoc block above explaining /tcp/ like /hub/ is explained?